src/utils/styleUtils.js

```diff
@@ -161,7 +161,7 @@ export const highlightPointStyle = new Style({
 // Change a width according to the change of DPI (from the old geoadmin)
 // Originally introduced here https://github.com/geoadmin/mf-geoadmin3/pull/3280
 export function adjustWidth(width, dpi) {
-    if (!width || isNaN(width) || width <= 0.0 || !dpi || isNaN(dpi) || dpi <= 0) {
+    if (!width || isNaN(width) || !dpi || isNaN(dpi) || dpi <= 0) {
```

Comment:
maybe @ltkum can better tell if this might break something or not...
I think that was a "trade-off" fix so that we would not have to add some fixes to the mapfishprint library or backend. I'll let him tell us what was the state on that

Reply:
Yes, I need @ltkum to the changes in this PR.

For this code, I remove it because for `offset`, it might be a negative number.

Reply:
From what I remember and can see : 

The `width <= 0.0` line was here because, afterwards, we use a math.max to set the scale size to its minimal value if it's smaller than it (as very small numbers would break the print). Currently, you don't check anymore if it's negative, which means it will set its offset to min scale when it's negative.

what I would do : 
```
function adjustWidth(width, dpi) {
if (!width || isNaN(width) || width <= 0.0 || !dpi || isNaN(dpi) || dpi <= 0) {return 0}
if (width <= 0) {return -adjustWidth(-width, dpi}
return Math.max((width * PRINT_DPI_COMPENSATION) / dpi, MIN_PRINT_SCALE_SIZE)
}
```

that way, your offsets are not set to 0, nor to MIN_PRINT_SCALE_SIZE